modules/govuk/manifests/apps/frontend.pp

```diff
@@ -33,14 +38,15 @@
   govuk::app { 'frontend':
     app_type               => 'rack',
     port                   => $port,
-    vhost_aliases          => ['private-frontend'],
+    vhost_aliases          => ['private-frontend', 'draft-frontend'],
```

Comment:
is this alias required if the default vhost is going to be set on the draft-frontend machines?